src/components/Card.js

```diff
@@ -34,19 +48,36 @@ export default class Card {
       this._handleImageClick(this._name, this._link);
     });
     this._likeButton.addEventListener("click", () => {
```

Comment:
Needs correction: All the listeners should be declared as the class methods, for example this method will describe the logic for toggling a like